ironfish/src/blockchain/blockchain.ts

```diff
@@ -552,15 +552,15 @@ export class Blockchain {
     return this.consensus.checkpoints.get(header.sequence)?.equals(header.hash) ?? false
   }
 
-  isEvmAsset(assetId: Buffer, tx?: BlockchainDBTransaction): Promise<boolean> {
-    return this.blockchainDb.assetIdToContract.has(assetId, tx)
+  async isEvmAsset(assetId: Buffer, tx?: BlockchainDBTransaction): Promise<boolean> {
+    return (await this.getEvmContractByAssetId(assetId, tx)) !== undefined
   }
 
   getEvmContractByAssetId(
     assetId: Buffer,
     tx?: BlockchainDBTransaction,
   ): Promise<Buffer | undefined> {
-    return this.blockchainDb.assetIdToContract.get(assetId, tx)
+    return this.blockchainDb.getContractTokenByAssetId(assetId, tx)
```

Comment:
do we need to extract the contract address from the returned buffer here?

Reply:
Good call, I will return extraced values